opencontrail_netns/instance_provisioner.py

```diff
@@ -46,22 +55,31 @@ def _virtual_network_lookup(self, network_name):
 
         return vnet
 
-    def vmi_locate(self, vm_instance, network, name, advertise_default=True):
-        fq_name = vm_instance.fq_name[:]
-        fq_name.append(name)
-        create = False
+    def project_lookup(self, fq_name):
         try:
-            vmi = self._client.virtual_machine_interface_read(fq_name=fq_name)
+            proj = self._client.project_read(fq_name=fq_name)
```

Comment:
project should be an argument and should not have the same name as the virtual-interface.
Both instance, network and vmi are children of project.